src/main/java/org/fisco/bcos/sdk/transaction/tools/ContractLoader.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2014-2020  [fisco-dev]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.fisco.bcos.sdk.transaction.tools;
+
+import org.apache.commons.lang3.tuple.Pair;
+
+/**
+ * ContractLoader
+ *
+ * @Description: ContractLoader
+ * @author maojiayu
+ * @data Jul 17, 2020 3:24:40 PM
+ *
+ */
+public class ContractLoader {
```

Comment:
Is the ContractLoader class should be in the module of Contract?

Reply:
It's based on the module of Contract